app/javascript/components/visualization/ScatterPlot.js

```diff
@@ -64,27 +64,10 @@ function RawScatterPlot({
 
       Plotly.react(graphElementId, scatter.data, layout)
 
-      const perfTimeFrontend = performance.now() - perfTimeFrontendStart
-
-      const perfTimeFull = perfTime + perfTimeFrontend
-
-      const perfLogProps = {
-        'perfTime:backend': perfTime, // Time for API call
-        'perfTime:frontend': Math.round(perfTimeFrontend), // Time from API call *end* to plot render end
-        'perfTime': Math.round(perfTimeFull), // Time from API call *start* to plot render end,
-        'numPoints': scatter.numPoints, // How many cells are we plotting?
-        genes,
-        'gene': scatter.gene,
-        'is3D': scatter.is3D,
-        'layout:width': width, // Pixel width of graph
-        'layout:height': height, // Pixel height of graph
-        'numAnnotSelections': scatter.annotParams.values.length,
-        'annotName': scatter.annotParams.name,
-        'annotType': scatter.annotParams.type,
-        'annotScope': scatter.annotParams.scope
-      }
-
-      log('plot:scatter', perfLogProps)
+      logScatterPlot(
+        { scatter, genes, width, height },
+        { perfTime, perfTimeFrontendStart }
```

Comment:
I still think this is a cleaner if you do 
```const perfTimeFrontend = performance.now() - perfTimeFrontendStart 
logScatterPlot(..., {perfTimeBackend, perfTimeFrontend})```

Reply:
That would give cleaner parameters, but add duplication (`perfTimeFrontend ` would also need to be calculated above `logViolinPlot`) and decrease coherence (the less `perfTime` calculation needed directly in app code, the better).

I'm preparing `perfTime` refinements for another PR, which I think will resolve both our concerns.